docs/builders/ephemeris/horizons-gui.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Download Horizons data via OpenSpace
+layout: default
+
+grand_parent: Builders
+parent: Ephemeris
+nav_order: 1
+---
+
+Since version 0.18.0 of OpenSpace there is a new feature that allows our users to access Horizons data directly via OpenSpace with no need to go to the [Horizons website](https://ssd.jpl.nasa.gov/horizons.cgi). This feature can be found in the OpenSpace launcher by pressing a series of buttons:
```

Comment:
"...directly via OpenSpace with no need to go to the [Horizons website]" -> "directly via OpenSpace, without needing to go to the [Horizons website]"